guide/integration.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 2
+---
+
+# Integrating with Fishjam Cloud
+
+Let's get familiar with how to integrate with Fishjam Cloud services.
+
+### Architecture diagram
+
+![Fishjam Cloud Data Flow](@site/static/img/architecture.svg)
+
+### Room manager
+
+To get you prototyping as fast as possible, the sandbox project comes with a HTTP server called **Room Manager**.  
+It allows you to start working on the Client App without having to setup an actual backend.  
+However, for production purposes, we recommend bringing your on backend and using our Server SDK's.
+
+#### How do I use it?
+
+Simply take the Fishjam instance url of your `sandbox` app, eg. `https://fishjam.io/api/v1/connect/***`, append a `/room-manager` path and use `roomName` and `participantName` query params to build an url for the GET request.
+
+##### Example url
+
+```
+https://fishjam.io/api/v1/connect/***/room-manager?roomName=foo&userName=bar
```

Comment:
**suggestion:** Maybe a placeholder for the token, just to make it clear?
```suggestion
https://fishjam.io/api/v1/connect/${YOUR_SERVER_TOKEN}/room-manager?roomName=foo&userName=bar
```

Reply:
very good idea, in fact it's just an app uuid, we can immediately mitigate confusion